Makefile

```
@@ -12,37 +12,21 @@ MODULES:=
 
 CLEAN:=
```

Define empty `BINS over here.
```suggestion
 CLEAN:=
 BINS:=
```